app/models/organism.rb

```diff
@@ -3,4 +3,11 @@ class Organism < ApplicationRecord
   has_many :infections
 
   validates :name, presence: true
+
+  # include PgSearch::Model
+  # pg_search_scope :search_by_name,
+  #                 against: :name,
```

Comment:
hi! I am not sure why we have a search by name here, while I thought we got rid of search by name for this iteration, considering species, age, temperature (and future location) those relevant for our scope

Reply:
@sabinacuccibar Yes, it was the only use case where it made sense to use the pg_search gem and I wanted to test it. I commented it out (instead of deleting it directly) in case we decide to use it at some point (but I also think that it doesn't make sense to search by name) or as an reminder if I figure out how to use the gem for the other search attributes.